.github/bors.toml

```diff
@@ -17,12 +17,35 @@ status = [
   "ubuntu (2.6.5, rubygems)",
   "ubuntu (2.6.5, bundler)",
```

Comment:
The regular CI matrix no longer has the `[rubygems, bundler]` dimension, so you should update all these names with `ubuntu (2.6.5)` and so on, I guess. Otherwise bundlerbot will just timeout I think.